ai_agent_apigen/fastapi_agent_scaffold/main.py

```diff
@@ -10,9 +10,19 @@
 from fastapi import UploadFile, File, Request
 from fastapi.responses import RedirectResponse
 import pandas as pd
-from ga_runner import run_ga_allocation
+from ai_agent_apigen.fastapi_agent_scaffold.ga_runner import run_ga_allocation
 from llm_assistant import generate_query_plan
 import json
+from pydantic import BaseModel
+
+class AllocationPayload(BaseModel):
+    academic: int
```

Comment:
Might be more durable if this is a float (i.e. new data where academic stuff isn't whole numbers)